src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -87,7 +87,7 @@ export function createNavLink(
     key: id,
     label: tooltip ?? title,
     href, // Use href and onClick to support "open in new tab" and SPA navigation in the same link
-    onClick(event) {
+    onClick(event: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
```

Comment:
Is this really needed? It seems already properly inferred on master

<img width="663" alt="Screenshot 2020-05-15 at 23 26 52" src="https://user-images.githubusercontent.com/1532934/82097679-a9f01100-9703-11ea-9349-3c0bfaa31b37.png">

Reply:
I can recheck this. When this was committed (last week), the `type_check` script was failing without the explicit type.

Reply:
You were correct; not needed.